subsystems/grippersubsystem.py

```diff
@@ -0,0 +1,97 @@
+# each roller alternates meaning if you were to pick up a cube you would eject a cone
+from enum import Enum, auto
+from commands2 import SubsystemBase
+from rev import SparkMaxLimitSwitch
+from wpilib import SmartDashboard
+
+from util.simneo import NEOBrushless
+
+import constants
+
+
+class GripperSubsystem(SubsystemBase):
+    class GripperState(Enum):
+        Intake = auto()  # Intake (wheels move right)
+        Outtake = auto()  # Outtake (wheels move left)
```

Comment:
the comment of direction is currently wrong as it will depend on cube and cone mode, comment can be omitted